pkg/ocm/client.go

```diff
@@ -64,13 +65,16 @@ func (l *SdKLogger) Error(ctx context.Context, format string, args ...interface{
 	l.FieldLogger.Errorf(format, args...)
 }
 
-func NewClient(config Config, log logrus.FieldLogger) (*Client, error) {
-
-	entry := log.(*logrus.Entry)
+func NewClient(config Config) (*Client, error) {
+	log := logrus.New()
```

Comment:
@ronniel1 @ErezAlster are we ok with having a separate logger for OCM? i.e. in order to support a different level than the global LOG_LEVEL.

Reply:
Why not pass a logger when creating the client?

Reply:
> Why not pass a logger when creating the client?
That's the way it was before this change :) but it means that we won't have granularity for different log levels, i.e. setting log level to debug would affect globally. In this case, we can simply set the LOG_LEVEL (instead of adding the new OCM_LOG_LEVEL). So if we're ok with not having control over the ocm log separately, then no need for this change.